boards/Makefile.common

```diff
@@ -168,6 +169,7 @@ endif
 SIZE      ?= $(TOOLCHAIN)-size
```

Comment:
Four lines above this we can use `RUSTC_SYSROOT` since we now have the variable.